SPECS/libseccomp/libseccomp.spec

```diff
@@ -48,7 +48,7 @@ find %{buildroot} -type f -name "*.la" -delete -print
 %files
 %license LICENSE
 %doc CREDITS README.md
-%{_libdir}/libseccomp.so.*
+%{_libdir}/libseccomp.so.2*
```

Comment:
Curious why we added a `2` explicitly.

Reply:
It's a way to immediately tell if the major version number for the shared library has changed. Such change is usually breaking and requires a re-build of all binaries linking against this library.

Example:

The package `moby-runc` has a BR on `libseccomp` and at least one of the binaries from `moby-runc` links dynamically against `libseccomp.so.2()(64bit)` (this dependency is generated automatically when the package is built and can be viewed if you run `rpm -qp --requires [moby-runc.rpm]` against the RPM or `dnf repoquery --requires moby-runc` from a CBL-Mariner image). If at some point we update the version of `libseccomp` to a one where the project owners have changed the major version number of the shared library, then with the previous wildcard we would have missed the update (the package would build without issues) and `moby-runc` would be stuck requiring the older version of the package. This way installing both `moby-runc` and the **latest** `libseccomp` would be impossible. And we want to install the latest most of the time, because most of our updates are for security reasons. With the way Thomas has changed the `%files` section we will break for a library version change and know that we should now also increment the release for packages with a BR on `libseccomp`, so that their binaries can link against the newer version of the shared library,

Reply:
+1 to what Pawel's comment says.

Reply:
@PawelWMS Thank you for the detailed explanation :)